Makefile

```diff
@@ -50,12 +103,40 @@ out/cypher-shell.zip: tmp/cypher-shell.zip
 	mkdir -p out
 	cp $< $@
 
-clean: ## Clean build directories
-	rm -rf out
-	rm -rf tmp
-	./gradlew clean
+out/rpm/SPECS/cypher-shell.spec: packaging/rpm/cypher-shell.spec
+	bash -c "mkdir -p out/rpm/{BUILD,RPMS,SOURCES,BUILDROOT,SPECS,SRPMS}/"
```

Comment:
Also seems surprising to do all the `mkdir`s here rather than in the individual targets below (as it already is for `BUILD`).